src/admin-views/flexible-tickets/ticket-types-column/types.php

```diff
@@ -0,0 +1,21 @@
+<?php
+/**
+ * The template for the ticket types column in the events list table.
+ *
+ * @since TBD
+ *
+ * @version TBD
+ *
+ * @var boolean $has_rsvp Whether the event has RSVP enabled.
+ * @var boolean $has_ticket Whether the event has Tickets enabled.
+ * @var boolean $has_series_pass Whether the event has Series Pass enabled.
+ * @var Tribe__Tickets__Admin__Views $admin_views The admin views instance for flexible tickets.
+ */
+?>
+<div class="tec-tickets__series_attached_ticket-types">
+	<?php
+		$admin_views->template( 'ticket-types-column/rsvp', [ 'has_rsvp' => $has_rsvp ] );
```

Comment:
I propose we change this code to take an array of tickets by ticket type.
This means there would be a template variable, e.g. `$tickets_by_type` that would be an array in this shape:
```
$tickets_by_type = [
  'rsvp' => [... RSVP Tickets... ],
  'default' => [... Tickets ... ],
  'series_pass' => [... Series Passes ... ],
];
```

Then, for each type, render the template for it:
```
foreach( $tickets_by_type as $type => $tickets){
  $admin_views->template( "ticket-types-column/$type", [ 'tickets' => $tickets ] );
}
```